files/en-us/web/javascript/reference/global_objects/array/flat/index.html

```diff
@@ -18,8 +18,8 @@
 
 <h2 id="Syntax">Syntax</h2>
 
-<pre
-  class="brush: js notranslate"><var>var newArray = arr</var>.flat(<em>[depth]</em>);</pre>
+<pre class="brush: js">let newArray = arr.flat();
+newArray = arr.flat(depth);</pre>
```

Comment:
I think syntax sections are meant to show the syntax (the method signature), and so it is not necessary to make this a a complete code example, in my opinion. The example section of the page is better suited for that. Syntax sections should allow very quick look up without much noise. They act as a reminder to how a method looks like.

Also on the plus side: we avoid deciding between var, let, const :)

```suggestion
<pre class="brush: js">flat()
flat(depth)</pre>
```

See also other pages that do that this way, like https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/copyWithin#syntax